### PR TITLE
refactor(webhooks): update fields in webhook events

### PR DESCRIPTION
**Pull request by daabr**
- Add `raw_url` alongside `url`
- Replace `body` functions with data (`bytes`, `form`, `json` - `None` if missing)
- Remove always-empty fields from `url`: leave only `path`, `query`, `fragment` (i.e. remove `scheme`, `opaque`, `host`)

Refs: ENG-1517, ENG-1518

### CODE REVIEW

File: internal/backend/webhookssvc/svc.go [hunk 1]
```diff
@@ -24,13 +24,13 @@ import (
 const WebhooksPathPrefix = "/webhooks/"
 
 type Service struct {
-	sl         *zap.SugaredLogger
+	logger     *zap.Logger
```
Comment (itayd) [inline internal/backend/webhookssvc/svc.go:27]: i use `l`, but no problem

File: internal/backend/webhookssvc/svc.go [hunk 2]
```diff
@@ -100,77 +105,73 @@ func requestToData(r *http.Request) (map[string]sdktypes.Value, error) {
...
+	return sdktypes.NewDictValueFromStringMap(
 		map[string]sdktypes.Value{
-			"text":  kittehs.Must1(sdktypes.NewConstFunctionValue("text", sdktypes.NewStringValue(string(body)))),
-			"bytes": kittehs.Must1(sdktypes.NewConstFunctionValue("bytes", sdktypes.NewBytesValue(body))),
-			"json":  jsonBody,
+			"fragment": sdktypes.NewStringValue(u.Fragment),
+			"path":     sdktypes.NewStringValue(u.Path),
+			"query": sdktypes.NewDictValueFromStringMap(
+				kittehs.TransformMapValues(u.Query(), func(vs []string) sdktypes.Value {
+					return sdktypes.NewStringValue(strings.Join(vs, ", "))
```
Comment (itayd) [inline internal/backend/webhookssvc/svc.go:169]:
```suggestion
					return sdktypes.NewStringValue(strings.Join(vs, ","))
```

**Review by itayd — APPROVED** [review]

**Comment by itayd** [comment]
i guess kittehub need to be updated as well?

File: internal/backend/webhookssvc/svc.go (hunk 2 above)
Comment (daabr) [reply internal/backend/webhookssvc/svc.go:169]: afacis, canonical implementations and examples of HTTP don't distinguish between:
```
Header: foo, bar
```
And:
```
Header: foo
Header: bar
```
Example: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept-Encoding

**Pull request merged**